.github/workflows/update_chains_preconfigured.yaml

```diff
@@ -0,0 +1,63 @@
+name: Update chains preconfigured
+
+on:
+  schedule:
+    - cron: '0 8 1/3 * *'
+  workflow_dispatch:
```

Comment:
```suggestion
  workflow_dispatch:
  pull_request:
```

Reply:
add here pull_request trigger in order to test it, that trigger will run every time when new commit is push to PR
https://docs.github.com/en/actions/writing-workflows/choosing-when-your-workflow-runs/events-that-trigger-workflows#pull_request